### ISSUE TITLE
`pnpm saas backend secrets` fails with  `unknown flag: --entrypoint /bin/bash`

### ISSUE THREAD

**Issue by matiasanaya** [apptension/saas-boilerplate#451]
### Describe the bug

I'm running `pnpm saas backend secrets` and I expect it to not fail, but instead if fails with:
```
unknown flag: --entrypoint /bin/bash
    Error: "docker compose run --rm --entrypoint /bin/bash ssm-editor /scripts/run.sh backend" failed with code 16
```

### Steps to reproduce

1. Run work through the [docs - Deploy to AWS](https://docs.demo.saas.apptoku.com/aws/deploy-to-aws/)
2. Run `npm saas backend secrets`

### System Info

```shell
$ npx envinfo --system --npmPackages  --binaries --browsers
sh: envinfo: command not found
```

... sorry.
```


### Logs

<details>
<summary>Click to expand!</summary>

```shell
pnpm saas backend secrets
----------
"dev" is set as a current environment stage. Live AWS session credentials are being used.
----------

Loaded environmental variables from SSM Parameter Store using chamber:
  service (prefix): dev
  count: 15

Settings secrets in AWS SSM Parameter store for:
  service: backend
  envStage: dev
  AWS account: 211125315823
  AWS region: us-east-1

pnpm nx run ssm-editor:compose-build-image

> nx run ssm-editor:compose-build-image

time="2024-01-08T21:41:40-03:00" level=warning msg="The \"SB_MIRROR_REPOSITORY\" variable is not set. Defaulting to a blank string."
time="2024-01-08T21:41:40-03:00" level=warning msg="The \"SB_PULL_THROUGH_CACHE_REPOSITORY\" variable is not set. Defaulting to a blank string."
time="2024-01-08T21:41:40-03:00" level=warning msg="The \"AWS_SECURITY_TOKEN\" variable is not set. Defaulting to a blank string."
time="2024-01-08T21:41:40-03:00" level=warning msg="The \"AWS_SESSION_EXPIRATION\" variable is not set. Defaulting to a blank string."
time="2024-01-08T21:41:40-03:00" level=warning msg="The \"SB_MIRROR_REPOSITORY\" variable is not set. Defaulting to a blank string."
time="2024-01-08T21:41:40-03:00" level=warning msg="The \"SB_PULL_THROUGH_CACHE_REPOSITORY\" variable is not set. Defaulting to a blank string."
#0 building with "desktop-linux" instance using docker driver
#1 [ssm-editor internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.0s
#2 [ssm-editor internal] load build definition from Dockerfile
#2 transferring dockerfile: 338B done
#2 DONE 0.0s
#3 [ssm-editor internal] load metadata for docker.io/segment/chamber:2
#3 DONE 1.2s
#4 [ssm-editor internal] load metadata for docker.io/library/alpine:latest
#4 DONE 2.2s
#5 [ssm-editor stage-1 1/6] FROM docker.io/library/alpine@sha256:51b67269f354137895d43f3b3d810bfacd3945438e94dc5ac55fdac340352f48
#5 DONE 0.0s
#6 [ssm-editor chamber 1/1] FROM docker.io/segment/chamber:2@sha256:6a707a98f50e3fa6afb1b957ffd5f6ba119a00571c87275edec068642a611f16
#6 DONE 0.0s
#7 [ssm-editor internal] load build context
#7 transferring context: 137B done
#7 DONE 0.0s
#8 [ssm-editor stage-1 5/6] COPY scripts /scripts
#8 CACHED
#9 [ssm-editor stage-1 2/6] RUN apk --no-cache add bash ca-certificates jq moreutils
#9 CACHED
#10 [ssm-editor stage-1 3/6] COPY --from=chamber /chamber /bin/chamber
#10 CACHED
#11 [ssm-editor stage-1 4/6] RUN mkdir -p /scripts
#11 CACHED
#12 [ssm-editor stage-1 6/6] RUN chmod a+x /scripts/*
#12 CACHED
#13 [ssm-editor] exporting to image
#13 exporting layers done
#13 writing image sha256:9d30f4ffb4ce3eb77207dbba1f4a700320ec003180af0e20f452857770086557 done
#13 naming to docker.io/library/saas-xero-bank-feed-ssm-editor done
#13 DONE 0.0s

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Successfully ran target compose-build-image for project ssm-editor (3s)

docker compose run --rm --entrypoint /bin/bash ssm-editor /scripts/run.sh backend
unknown flag: --entrypoint /bin/bash
    Error: "docker compose run --rm --entrypoint /bin/bash ssm-editor /scripts/run.sh backend" failed with code 16
```
</details>

### Validations

- [X] Follow our [Code of Conduct](https://github.com/apptension/saas-boilerplate/blob/master/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/apptension/saas-boilerplate/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://docs.demo.saas.apptoku.com/).
- [X] Check that there isn't [already an issue](https://github.com/apptension/saas-boilerplate/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/apptension/saas-boilerplate/discussions) or join our [Discord Chat Server](https://discord.apptension.com).

**Comment by pziemkowski** [comment]
Can you let me know what system you are working on and which version of docker you have?

We use this command a lot and it worked so far so it might be some difference in our versions. Need to make sure both still work.

**Comment by matiasanaya** [comment]
Hey, I'm on `MacOS 14.2.1` and running `Docker Desktop 4.26.1 (131620)`.

**Comment by Crunchyman-ralph** [comment]
I fixed it like this:
```diff
await runCommand(
    'docker',
    [
      'compose',
      'run',
      '--rm',
-    '--entrypoint /bin/bash',
+   '--entrypoint',
+   '/bin/bash',
     'ssm-editor',
     `/scripts/run.sh`,
     serviceName,
    ],
    {
      cwd: rootPath,
    },
  );
```

**Comment by matiasanaya** [comment]
> I fixed it like this:
> 
> ```diff
> await runCommand(
>     'docker',
>     [
>       'compose',
>       'run',
>       '--rm',
> -    '--entrypoint /bin/bash',
> +   '--entrypoint',
> +   '/bin/bash',
>      'ssm-editor',
>      `/scripts/run.sh`,
>      serviceName,
>     ],
>     {
>       cwd: rootPath,
>     },
>   );
> ```

@Crunchyman-ralph that does the trick for the flag issue, though you'll probably still need to modify `/scripts/run.sh` since `serviceName` is `$1` and not `$2`, for the command to actually do it's job. Still I think the linked PR (#452) is better, because the `ssm-editor` image already sets the entrypoint to `/scripts/run.sh`

**Comment by mkleszcz** [comment]
It looks like this issue occur on some newer docker version.
Merged #452
